views/index.ejs

```diff
@@ -67,79 +68,128 @@
+
+<div class="container-fluid">
+  
+        
+  <div class="row">
+    <div class="col-sm-2 "   style="background-color:#d2b481;position:fixed">
+    <div>
+    <br><br><br><br><br><br><br>
+    <div style="padding-left:30px">
+      <p style="color:white;font-family:Arial;font-size:30px">   Medicos</p>
+      <p style="color:grey;font-family:Arial;font-size:18px;padding-left:20px">   To infinity</p>
+      <img src="http://valleyosteopathy.com.au/wp-content/uploads/2013/11/facebook-default-no-profile-pic1.jpg" alt="ide" height="100" width="100" class="img-circle"></img></div>
+      <br><button type="button" class="btn btn-warning btn-sm" style="margin-left:45px ">Logout</button>
+<br><br><br><br>
+</div>
+<div class="sidebar-nav" style="background-color:#d2b481;padding:0px 0px 0px 0px;margin:0px 0px 0px 0px;border:1px #d2b481 ">
+      <div class="navbar navbar-default" role="navigation" style="background-color:#d2b481;border:1px #d2b481">
+        <div class="navbar-header" style="background-color:#d2b481;border:1px #d2b481">
+          
+          <span class="visible-xs navbar-brand">Sidebar menu</span>
+        </div>
+        <div class="navbar-collapse collapse sidebar-navbar-collapse" style="background-color:#d2b481">
+          <ul class="nav navbar-nav" style="background-color:#d2b481">
+            <li class="active" style="background-color:red"><a style="background-color:#d2b481"href="#">Appointments</a></li>
+            <li style="background-color:#d2b481"><a href="#" ng-click="goToThisDetails('doctors',true)">Doctors</a></li><br><br><br><br>
+            <li style="background-color:#d2b481;padding-left:3px;margin-left:1px"><a href="#" ng-click= "goToThisDetails('meetings',true)">Meetings</a></li>
+            <li style="background-color:#d2b481"><a href="#"  ng-click="goToThisDetails('prescriptions',true)" >Prescriptions</a></li>
+            <li style="background-color:#d2b481"><a href="#" align="bottom" ng-click="goToThisDetails('personalinfo',true)" >Personal Information</a></li>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+            <li></li>
+
+
+            
+          </ul>
+        </div><!--/.nav-collapse -->
       </div>
     </div>
+
+
+
+
+    </div>
+    <div class="col-sm-10 col-lg-offset-2 "   style="background-color:grey;">
+    <br> <br> <br><br><br><br>
+      <p style="font-size:20px; padding-left:50px"> Medicos helps you connect with your psychotherapist easily,whenever,wherever.</p>
+      <p style="font-size:20px ;padding-left:50px"><a href="#">Read articles</a>,<a href="#">take part in discussions</a><a href="#"> and become a part of the community</a>
+
+
+      <div class="jumbotron">
+      <h1>Welcome {{currentUser()}}</h1>
+      <br>
+      <button type="button" class="btn btn-warning" style="margin-left:30px">See Appointments</button>
+    </div>
+
+<br><br><br><br><br><br><br><br><br><br><br><br>
+
+<div id="doctors">
+<div class="jumbotron">
```

Comment:
Add Space . Formatting is very important so that others can understand faster. :)